src/DetectorConstruction.cpp

```diff
@@ -18,18 +19,19 @@
 #include "G4SDManager.hh"
 #include "G4LogicalVolumeStore.hh"
 
-
 DetectorConstruction::DetectorConstruction(): G4VUserDetectorConstruction()
-{}
+{
+}
 
 DetectorConstruction::~DetectorConstruction()
-{}
+{
+}
 
 G4VPhysicalVolume* DetectorConstruction::Construct()
 {
   // WORLD /////////////////////////////////////////////////
 
-  G4double world_size = 15.*m;
+  G4double world_size = 30.*m;
```

Comment:
Should this be a configuranble parameter?  Or maybe computed from the max of the three detector dimensions?

Reply:
probably could at some point, but I dont see this being changed at all

Reply:
fixing in bugfix/v3.0.0_revisions branch